pkg/pool-weighted/contracts/lib/CircuitBreakerLib.sol

```diff
@@ -252,104 +53,12 @@ library CircuitBreakerLib {
     }
 
     /**
-     * @notice Sets the reference BPT price, weight complement, and upper and lower bounds for a token.
-     * @dev If a bound is zero, it means there is no circuit breaker in that direction for the given token.
-     * @param bptPrice: The BPT price of the token at the time the circuit breaker is set. The BPT Price
-     *   of a token is generally given by: supply * weight / balance.
-     * @param weightComplement: This is (1 - currentWeight).
-     * @param lowerBound: The value of the lower bound, expressed as a percentage.
-     * @param upperBound: The value of the upper bound, expressed as a percentage.
+     * @notice Convert bounds to BPT prices
```

Comment:
Updated the comment. It gets called several places in the storage lib where it needs to have ratio outputs (so it can store them).

What I can do is create a `calcBptPriceBoundaries`, and then also call it below.